app/apollo/test/channel.fga.spec.js

```diff
@@ -0,0 +1,666 @@
+      });
+
+      expect(result.data.data.addChannel.uuid).to.be.an('string');
+      const channel1 = await models.Channel.findOne({uuid: result.data.data.addChannel.uuid});
+      expect(channel1.data_location).to.equal('dal');
+
+    } catch (error) {
+      if (error.response) {
+        console.error('error encountered:  ', error.response.data);
+      } else {
+        console.error('error encountered:  ', error);
+      }
+      throw error;
+    }
+  });
+
+  // addChannel without authentication
+  it('fgaUser01 does NOT have authentication to add a channel', async () => {
+    try {
+      const result = await channelApi.addChannel(fgaToken01, {
+        orgId: org01._id,
+        name: 'testConfiguration3',
+        data_location: 'dal',
+      });
+
+      expect(result.data.data).to.equal(null);
+
+    } catch (error) {
+      if (error.response) {
+        console.error('error encountered:  ', error.response.data);
+      } else {
+        console.error('error encountered:  ', error);
+      }
+      throw error;
+    }
+  });
+
+  // editChannel
+  it('fgaUser01 has authentication to edit channel and update channel name', async () => {
+    try {
+      const result = await channelApi.editChannel(fgaToken01, {
+        orgId: org01._id,
+        uuid: 'testConfiguration1',
+        name: `${'test-configuration1'}_new`
+      });
+      console.log( `editChannel result: ${JSON.stringify( result.data, null, 2 )}` );
+      const editChannel = result.data.data.editChannel;
+
+      expect(editChannel.success).to.equal(true);
+      expect(editChannel.name).to.equal(`${'test-configuration1'}_new`);
+
+    } catch(error) {
+      if (error.response) {
+        console.error('error encountered:  ', error.response.data);
+      } else {
+        console.error('error encountered:  ', error);
+      }
+      throw error;
+    }
+  });
+
+  // editChannel without authentication
+  it('fgaUser01 does NOT have authentication to edit channel and update channel name', async () => {
+    try {
+      const result = await channelApi.editChannel(fgaToken01, {
+        orgId: org01._id,
+        uuid: 'testConfiguration2',
+        name: `${'test-configuration2'}_new`
+      });
+
+      expect(result.data.data).to.equal(null);
+
+    } catch(error) {
+      if (error.response) {
+        console.error('error encountered:  ', error.response.data);
+      } else {
+        console.error('error encountered:  ', error);
+      }
+      throw error;
+    }
+  });
+
+  // removeChannel
+  it('fgaUser01 has authentication to remove channel', async () => {
+    try {
+      const result2 = await channelApi.removeChannel(fgaToken01, {
+        orgId: org01._id,
+        uuid: 'testConfiguration1',
+      });
+      const removeChannel = result2.data.data.removeChannel;
+
+      expect(removeChannel.success).to.equal(true);
+      expect(removeChannel.uuid).to.equal('testConfiguration1');
+
+      // step 3 validate the channel is not there
```

Comment:
This isn't a "step 3" it's a "step 2".  Was the intent that this test would (1) create a new channel, (2) delete the newly created channel, then (3) verify the channel is removed?

Reply:
I forgot to add a comment and arrange them properly. All "step" comments have been revisited.

Reply:
Resolved